vitest.config.ts

```diff
@@ -0,0 +1,8 @@
+/// <reference types="vitest/config" />
+import { defineConfig } from 'vitest/config';
```

Comment:
nit: the reference to `vitest/config` is only necessary when importing `defineConfig` from `vite`, we don't need it when we're already importing that type directly:

```suggestion
import { defineConfig } from 'vitest/config';
```

Reply:
Ah, that makes sense. I also thought that was odd, but I think @erickzhao and I got confused by the docs, see https://github.com/electron/forge/pull/3797/files#r1909540910. The docs link in those comments is dead now, but I see what you're referring to [in the current docs](https://vitest.dev/guide/#configuring-vitest). Good to know going forward, we'll clean them up. Thanks @erikian!